src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -139,14 +146,16 @@ public async Task Build_params_with_correct_overrides_succeeds_with_values_overr
                         otherProp: "otherValue"
                     }
                 }
-                """;    
+                """;
 
-            Environment.SetEnvironmentVariable("BICEP_PARAMETERS_OVERRIDES", paramsOverrides);
```

Comment:
Since each of the test now depends on an in-memory version of the environment, should we remove the `[DoNotParallelize]` decorator?

Reply:
Good point, done!